models/Sauce.js

```diff
@@ -0,0 +1,19 @@
+// Importation de Mongoose
+const mongoose = require("mongoose");
+
+// Création d'un schéma de données (informations requises)
+const sauceSchema = mongoose.Schema({
+  name: { type: String, required: true },
```

Comment:
Missing userId

Reply:
Maybe I misunderstand, but I learned that "userId" was automatically generated by MongoDB (database), and therefore it was not necessary to add it in the model. (that's why it was missing).